client/src/components/backend/Drawer/Wrapper.js

```diff
@@ -169,4 +188,9 @@ class DrawerWrapper extends PureComponent {
   }
 }
 
+DrawerWrapper.childContextTypes = {
```

Comment:
Can't these be added as static properties on the class? That would be more in keeping with what we're doing elsewhere with propTypes, etc.